engine/src/main/java/org/camunda/bpm/engine/authorization/HistoricTaskPermissions.java

```diff
@@ -40,7 +40,10 @@
   ALL("ALL", Integer.MAX_VALUE),
 
   /** Indicates that READ interactions are permitted. */
-  READ("READ", 2);
+  READ("READ", 2),
+
+  /** Indicates that READ_VARIABLE interactions are permitted. */
+  READ_VARIABLE("READ_VARIABLE", 4);
```

Comment:
note to myself, double check the value

Reply:
we agreed to put 64 as value for `READ_VARIABLE` permission to be consistent with `TaskPermissions#READ_VARIABLE`

Reply:
```suggestion
  READ_VARIABLE("READ_VARIABLE", 64);
```